assets/app/views/directives/_status-icon.html

```diff
@@ -1,4 +1,4 @@
-<span ng-switch="status" class="hide-ng-leave status-icon">
+<span ng-switch="status | capitalize" class="hide-ng-leave status-icon">
```

Comment:
I have a bit of concern about this.... seems like we should be able to see `Pending`, `pending`, `PENDING`, `PeNdInG`, etc all as 'pending'.  I'd be concerned if there is a difference in cases that we need to know about.  

Maybe its clearer when we talk about `CrashLoopBackOff`, `crashloopbackoff`, `CRASHLOOPBACKOFF`, etc. since that is actually multiple words, but it still seems like all options should convey the exact same concept.  

In a sense, `status-icon` is kind of acting as documentation when it has to be an exact match...hmm.

Thoughts?